### ISSUE TITLE
Show linter error when I am on the error line

### ISSUE THREAD

**Issue by evandrocoan** [DamnWidget/anaconda#688]
## Expected Behaviour

Show linter error when I am on the error lines like:

![image](https://user-images.githubusercontent.com/5332158/29222266-2b47acd6-7e98-11e7-9d1d-c49174153f36.png)

## Actual Behaviour

You can only see the linter  marks, but not error information. So you have no clue on what is wrong.

## Steps to Reproduce

Create the file:
```
if 'LOG_FILE_NAME' in globals():

    # Clear the log file contents
    open(LOG_FILE_NAME, 'w').close()
```
![image](https://user-images.githubusercontent.com/5332158/29222334-6dcdf876-7e98-11e7-9ed4-6e636fb170ce.png)

## Anaconda's JsonServer Logs
**Note**: Anaconda's JsonServer logs can be found in:
* Linux: ~/.local/share/anaconda/logs
* OS X: ~/Library/Logs/anaconda
* Windows: %APPDATA\\Anaconda\\Logs

**Comment by DamnWidget** [comment]
No, you don't have to guess the errors, that would be beyond stupid. Actually you have four ways to know about the errors.

- Method 1: Just select the offending line and an error description will appear in the status bar
- Method 2: Use the Command Palette to execute: `Anaconda: Show error list`
- Method 3: Set the `anaconda_linter_show_errors_on_save` option to `true` in the config
- Method 4: Set the `anaconda_linter_phantoms` option to `true` to see errors inline

As you can see, you don't have to guess too much, have you?

**Comment by evandrocoan** [comment]
Ok, however the phantoms always show up. Which is horrible. They show only appear when I select the line or hovers it. Following this, the Sublime Text pop up could be used instead of phantoms:

![image](https://user-images.githubusercontent.com/5332158/29227298-bd10153c-7eab-11e7-85a3-434cd76d5886.png)

**Comment by DamnWidget** [comment]
Then this is a feature request and not an issue, not a support request, honestly I think there are already more than enough ways to get information about the linting problems, apart from that, new features are closed till I release Golconda and I can not use the already low spare time that I have implementing and supporting new features.

Said that, I will accept a PR implementing this.

**Comment by enuchi** [comment]
I know this is an old thread but just seeing if anyone here worked on this. This would be a useful feature.

**Comment by evandrocoan** [comment]
I fixed this by using https://github.com/palantir/python-language-server with https://github.com/tomv564/LSP which already has this implemented.

**Comment by bitomaxsp** [comment]
No wonder i ended up here. Still not solved in 2020 :)

**Comment by nguyenbathanh** [comment]
I decided to use [SublimeLinter](http://www.sublimelinter.com/en/stable/) and disable Anaconda linting.

![image](https://user-images.githubusercontent.com/4046112/124214947-4ac6a900-db1d-11eb-972a-19fa51e9c251.png)
